### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added scenario to address credential prompt issue

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ description: "Summary: Use the steps in this article to synchronize public folde
...
 
-This topic describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
+This topic describes how to configure EXO mailboxes to access public folders at On-Premises. The topic also describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:28]:
```suggestion
This topic describes how to configure EXO mailboxes to access on-premises public folders. The topic also describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ You will sync your mail-enabled public folders by using the following scripts, w
 
 - `SyncMailPublicFolders.strings.psd1`: This is a support file used by the preceding synchronization script and should be copied to the same location as the preceding script.
 
+> [!NOTE]
+> The `Sync-MailPublicFolders.ps1` does not support modern authentication and works only with basic authentication. You must use an EXO account that supports basic authentication.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:40]:
```suggestion
> The `Sync-MailPublicFolders.ps1` does not support modern authentication and works only with basic authentication. You must use an Exchange Online account that supports basic authentication.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 3]
```diff
@@ -116,15 +120,15 @@ A hybrid configuration with Exchange 2003 public folders is not supported. If yo
 
 5. Repeat the preceding steps for every public folder server in your organization.
 
-## Step 3: Download the scripts
+## Step 3: Download the script to synchronize mail enabled public folders
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:123]:
```suggestion
## Step 3: Download the script to synchronize mail-enabled public folders
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 4]
```diff
@@ -116,15 +120,15 @@ A hybrid configuration with Exchange 2003 public folders is not supported. If yo
...
 
 1. Download the following files from [Mail-enabled Public Folders - directory sync script](https://www.microsoft.com/download/details.aspx?id=46381):
 
    - `Sync-MailPublicFolders.ps1`
 
    - `SyncMailPublicFolders.strings.psd1`
 
-2. Save the files to the local computer on which you'll be running PowerShell. For example, C:\PFScripts.
+2. Save the files to Exchange On-Prmises server on which you'll be running the script. For example, C:\PFScripts.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:131]:
```suggestion
2. Save the files to the Exchange on-premises server on which you'll be running the script. For example, C:\PFScripts.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 5]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
 
    Where `Credential` is your Microsoft 365 or Office 365 user name and password, and `CsvSummaryFile` is the path to where you would like to log synchronization operations and errors, in .csv format.
 
+> [!NOTE]
+> The `Sync-MailPublicFolders.ps1` does not support modern authentication and works only with basic authentication. You must use an EXO account that supports basic authentication.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:149]:
```suggestion
> `Sync-MailPublicFolders.ps1` does not support modern authentication and works only with basic authentication. You must use an Exchange Online account that supports basic authentication.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 6]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
...
+
+
 > [!NOTE]
 > Before running the script, we recommend that you first simulate the actions that the script would take in your environment by running it as described above with the `-WhatIf` switch. <br/> We also recommend that you run this script daily to synchronize your mail-enabled public folders.
 
 ## Step 5: Configure Exchange Online users to access on-premises public folders
 
+A Microsoft 365 or Office 365 mailbox who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:157]:
```suggestion
A Microsoft 365 or Office 365 mailbox that is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 7]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
...
+
+
 > [!NOTE]
 > Before running the script, we recommend that you first simulate the actions that the script would take in your environment by running it as described above with the `-WhatIf` switch. <br/> We also recommend that you run this script daily to synchronize your mail-enabled public folders.
 
 ## Step 5: Configure Exchange Online users to access on-premises public folders
 
+A Microsoft 365 or Office 365 mailbox who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  
+
+You can use following command to identify such mailboxes:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:159]:
```suggestion
You can use the following command to identify such mailboxes:
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 8]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
...
+
+```PowerShell
+Get-Mailbox |?{$_.IsDirSynced -eq $false}
+```
+
+The above users will keep getting credential prompt after public folder mailboox access is configured. Use one of the following solution for such users, before enabling public folder access:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:165]:
```suggestion
These users will keep getting credential prompts after public folder mailbox access is configured. Use one of the following solutions for such users before enabling public folder access:
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 9]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
...
+
+1) Link the EXO only mailboxes, listed in step above, to On-Premises users by using the Knowledge Base article [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:167]:
```suggestion
1. Link only the Exchange Online mailboxes listed in the previous step to on-premises users as described in [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 10]
```diff
@@ -141,11 +145,29 @@ The Directory Synchronization service doesn't synchronize mail-enabled public fo
...
+
+2) Use steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users at On-Premises.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:169]:
```suggestion
2. Use the steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users on-premises.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — COMMENTED** [review]
@Batre-MSFT - please also commit the minor suggested change to line 159. Thanks.

**Comment by Batre-MSFT** [comment]
> @Batre-MSFT - please also commit the minor suggested change to line 159. Thanks.

I hope all changes are committed now

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md [hunk 11]
```diff
@@ -25,7 +25,7 @@ description: "Summary: Use the steps in this article to synchronize public folde
...
 
-This topic describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
+This topic describes how to configure EXO mailboxes to access on-premises public folders. The topic also describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-legacy-hybrid-public-folders.md:28]:
```suggestion
This topic describes how to configure Exchange Online mailboxes to access on-premises public folders. The topic also describes how to synchronize mail-enabled public folders if your users are in Microsoft 365 or Office 365 and your Exchange Server 2010 SP3 public folders are on-premises. However, a Microsoft 365 or Office 365 user who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access legacy or modern on-premises public folders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@Batre-MSFT - my apologies, please also commit this change that I have missed. Many thanks for committing all the others. Looks good.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @Batre-MSFT.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**